flatbuffers/flatbuffers_case_study.md

```diff
@@ -0,0 +1,151 @@
+Case study of Javascript Flat Buffers
+================================
+
+Data serialization is very common process in web application. So common that no one thinks of it. You just pull data from the DB, and your server and client frameworks just handle this thing for you. The server serializes the data into JSON, and the client parses it to JS object. This flow is simple and natural but, as everything, when taken to the extreme start to show scaling issues.
```

Comment:
the first sentence reads weird..
maybe:
Data serialization is a very common process in web applications